tests/data/Vault.vy

```diff
@@ -852,3 +852,889 @@ def sweep(_token: address):
     # Can't be used to steal what this Vault is protecting
     assert _token != self.token.address
     self.erc20_safe_transfer(_token, self.governance, ERC20(_token).balanceOf(self))
+
+
+# output
+# @version 0.3.3
+
+API_VERSION: constant(String[28]) = "0.1.2"
+
+# TODO: Add ETH Configuration
+# TODO: Add Delegated Configuration
+from vyper.interfaces import ERC20
+
+implements: ERC20
+
+interface DetailedERC20:
+
+    def name() -> String[42]:
+        view
+
+    def symbol() -> String[20]:
+        view
+
+    def decimals() -> uint256:
+        view
```

Comment:
Would it be possible to get it to render like this? it's a lot easier to read this way
```suggestion
interface DetailedERC20:
    def name() -> String[42]: view
    def symbol() -> String[20]: view
    def decimals() -> uint256: view
```

Reply:
Done.